tasks/src/main/resources/application.conf

```diff
@@ -18,3 +18,31 @@
 taskRunner {
   tasks = ${?TASKS}
 }
+
+configManager {
+  secretProvider = ${?TASKS_SECRET_PROVIDER}
+  configSecretFileList = ${?TASKS_SECRET_FILES}
+  allowSecretsFromConfig = ${?ALLOW_SECRETS_FROM_CONFIG}
+}
+
+reportStorage {
+  name = "database"
+  name = ${?REPORT_STORAGE_NAME}
+  namespace = "reports"
+  namespace = ${?REPORT_STORAGE_NAMESPACE}
+  inMemoryLimit = 1048576
+  inMemoryLimit = ${?REPORT_STORAGE_IN_MEMORY_LIMIT}
+  azureBlobContainerName = ${?REPORT_STORAGE_CONTAINER_NAME}
+  azureBlobEndpointUrl = ${?REPORT_STORAGE_ENDPOINT_URL}
+  azureBlobStorageAccountName = ${?REPORT_STORAGE_STORAGE_ACCOUNT_NAME}
+  s3AccessKey = ${?REPORT_STORAGE_ACCESS_KEY}
+  s3SecretKey = ${?REPORT_STORAGE_SECRET_KEY}
+  s3Region = ${?REPORT_STORAGE_REGION}
+  s3BucketName = ${?REPORT_STORAGE_BUCKET_NAME}
+  s3EndpointUrl = ${?REPORT_STORAGE_ENDPOINT_URL}
+}
+
+dataRetention {
+  ortRunDays = 90
```

Comment:
Is this the default that#s now in place? I'm not sure that number would be fine with us. Of course, we could configure it in our deployment. But I'm wonder whether the feature as a whole should be disabled instead by default to not change behavior compared to before. Only users of the server who want / need it should enabled it. Is there even a way to disable the feature completely now, e.g. by setting the number of days to a negative value?

Same for other maintenance / data retention task introduced recently / to be introduced. What do you think?

Reply:
The way the maintenance tasks are implemented currently (refer to the corresponding [discussion](https://github.com/eclipse-apoapsis/ort-server/discussions/1560)), they are not active automatically, but users have to manually schedule them using an external mechanism. In our environment, we will configure a Kubernetes Cron job to run this task periodically. If you do not do this, you don't have to worry that runs are suddenly deleted.